### PR TITLE
Commerce 8309 Order detail does not display the related Account

### PR DESCRIPTION
**Pull request by christian-chiappa**

### CODE REVIEW

**Comment by christian-chiappa** [comment]
ci:test:relevant

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]

File: modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderEditDisplayContext.java [hunk 1]
```diff
@@ -463,6 +463,20 @@ public BigDecimal getDecimalQuantity(CommerceOrderItem commerceOrderItem) {
...
+	public String getDescriptiveAccountInfo(CommerceAccount commerceAccount) {
+		if (commerceAccount == null) {
+			return StringPool.BLANK;
+		}
+
+		StringBundler sb = new StringBundler(3);
+
+		sb.append(commerceAccount.getName());
+		sb.append(" #");
+		sb.append(commerceAccount.getCompanyId());
```
[the remark this answers is not in the document]
Comment (christian-chiappa) [reply modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderEditDisplayContext.java]: fixed

File: modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderEditDisplayContext.java [hunk 2]
```diff
@@ -463,6 +463,21 @@ public BigDecimal getDecimalQuantity(CommerceOrderItem commerceOrderItem) {
...
 
+	public String getDescriptiveAccountInfo(CommerceAccount commerceAccount) {
+		if (commerceAccount == null) {
+			return StringPool.BLANK;
+		}
+
+		StringBundler sb = new StringBundler(4);
+
+		sb.append(commerceAccount.getName());
+		sb.append(StringPool.COMMA_AND_SPACE);
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-order-web/src/main/java/com/liferay/commerce/order/web/internal/display/context/CommerceOrderEditDisplayContext.java:474]: Looking at the wireframes I see the ID in a new line under the account name.
I guess you should put a
sb.append(StringPool.NEW_LINE);
if you want to use this code.

**Review by luca-pellizzon — CHANGES_REQUESTED** [review]

**Comment by christian-chiappa** [comment]
ci:test:relevant

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**